packages/geo/src/lib/directions/shared/directions.utils.ts

```diff
@@ -432,6 +432,7 @@ export function formatInstruction(
   languageService: LanguageService,
   lastStep = false
 ) {
+  route = route ? route : 'Voie';
```

Comment:
On pourrait inclure la valeur par défaut dans les paramètres. On pourrait aussi ajouter le type tant qu'a y être
```suggestion
export function formatInstruction(
  type,
  modifier,
  route: string = 'Voie',
  direction,
  stepPosition,
  exit,
  languageService: LanguageService,
  lastStep = false
) {
```